mdal/frmts/mdal_2dm.hpp

```diff
@@ -13,6 +13,7 @@
 #include "mdal_memory_data_model.hpp"
 #include "mdal.h"
 #include "mdal_driver.hpp"
+#include "mdal_logger.hpp"
```

Comment:
do you need it in hpp file? move include in cpp file (everywhere where you do not need anything from logger in the header., if you do anything in general is better to use forward declaration to speed up compilation times)